style.css

```diff
@@ -204,6 +204,64 @@ body {
   z-index: 100;
 }
 
+/* mobile menu */
+
+.visible {
+  display: flex;
+}
+
+.non-visible {
+  display: none;
+}
+
+.nav-menu {
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+  background-color: #0113d6;
+  color: white;
+  height: 812px;
+  padding: 5px;
+  opacity: 65%;
+  transition: all;
+  transition-duration: 0.75s;
+  backface-visibility: hidden;
+}

```

Comment:
- [ ] As requested by the previous reviewer, The mobile menu should cover the whole height, Currently at the bottom I can see the page content. Please make sure to fix this issue.

![image](https://user-images.githubusercontent.com/83803414/232571752-552e3d42-d4ba-4522-9d54-f8210c8163c6.png)

Reply:
### Okay I am so sorry I forgot it, wait a moment  I will fix it in a short time.